docs/plugins/verifier/notation.md

```diff
@@ -97,7 +103,7 @@ spec:
 ```
 
 In the example, the verifier's configuration references 2 `KeyManagementProvider`s, kmp-akv-ca, kmp-akv-tsa. Here, `ca:ca-certs` is one of the trust stores specifing and the `ca-certs` suffix corresponds to the `ca-certs` certificate collection listed in the `verificationCertStores` section.
-To use the Time-stamping feature users need to config trust store type accordingly. In the sample, `tsa:tsa-certs` is the trust stores specifing for Time-stamping and the `tsa-certs` suffix corresponds to the `tsa-certs` certificate collection listed in the `verificationCertStores` section.
+To use timestamping feature users need to config trust store type accordingly. In the sample, `tsa:tsa-certs` is the trust stores specifing for timestamping and the `tsa-certs` suffix corresponds to the `tsa-certs` certificate collection listed in the `verificationCertStores` section.
```

Comment:
```suggestion
To use the timestamping feature, you need to configure the trust store type accordingly. In the example, `tsa:tsa-certs` is the trust store specified for timestamp verification, and the `tsa-certs` suffix corresponds to the `tsa-certs` certificate collection listed in the `verificationCertStores` field.
```

Reply:
It seems we are coupling the timestamp certificate verification and CA certificate verification; it may mislead users that timestamp certificate verification is a required configuration and complex the getting started experience. I think we may need to split the sample configuration by different scenarios, e.g. testing and prod. 

For example, Notation has a separate doc to walk through timestamp feature and configuration in https://v1-2.notaryproject.dev/docs/user-guides/how-to/timestamping/

Reply:
As discussed, will open a new PR for adding more details about TSA